src/overlays/actors/ovl_En_Go/z_en_go.h

```diff
@@ -2,105 +2,127 @@
 #define Z_EN_GO_H
 
 #include "global.h"
+#include "objects/object_oF1d_map/object_oF1d_map.h"
 
 struct EnGo;
 
 typedef void (*EnGoActionFunc)(struct EnGo*, PlayState*);
 
-#define ENGO_GET_F(thisx) (((thisx)->params & 0xF) & 0xFF)
-#define ENGO_GET_70(thisx) ((((thisx)->params & 0x70) >> 4) & 0xFF)
-#define ENGO_GET_7F80(thisx) ((((thisx)->params & 0x7F80) >> 7) & 0xFF)
+#define ENGO_GET_TYPE(thisx) (((thisx)->params & 0xF) & 0xFF)
+#define ENGO_GET_SUBTYPE(thisx) ((((thisx)->params & 0x70) >> 4) & 0xFF)
 
-enum {
-    /* 0 */ ENGO_F_0,
-    /* 1 */ ENGO_F_1,
-    /* 2 */ ENGO_F_2,
-    /* 3 */ ENGO_F_3,
-    /* 4 */ ENGO_F_4,
-    /* 5 */ ENGO_F_5,
-    /* 6 */ ENGO_F_6,
-    /* 7 */ ENGO_F_7,
-    /* 8 */ ENGO_F_8,
-};
+#define ENGO_GET_PATH(thisx) ((((thisx)->params & 0x7F80) >> 7) & 0xFF)
 
-enum {
-    /* 0 */ ENGO_70_0,
-    /* 1 */ ENGO_70_1,
-    /* 2 */ ENGO_70_2,
-    /* 3 */ ENGO_70_3,
-    /* 4 */ ENGO_70_4,
-    /* 5 */ ENGO_70_5,
-};
+#define ENGO_NUM_SNOW_EFFECTS 16
+#define ENGO_NUM_EFFECTS 32
 
-typedef struct {
-    /* 0x00 */ u8 unk_00;
-    /* 0x01 */ u8 unk_01;
-    /* 0x02 */ u8 unk_02;
-    /* 0x04 */ Vec3s unk_04;
-    /* 0x0A */ Vec3s unk_0A;
-    /* 0x10 */ Vec3f unk_10;
-    /* 0x1C */ Vec3f unk_1C;
-    /* 0x28 */ Vec3f unk_28;
-    /* 0x34 */ f32 unk_34;
-    /* 0x38 */ f32 unk_38;
-} EnGoStruct; // size = 0x3C;
+extern const ActorInit En_Go_InitVars;
+
+typedef struct EnGoEffect {
+    /* 0x00 */ u8 effectType;
+    /* 0x01 */ u8 alphaDenom;
+    /* 0x02 */ u8 alphaNumer;
+    /* 0x04 */ Vec3s rotVelocity;
+    /* 0x0A */ Vec3s rotAngle;
+    /* 0x10 */ Vec3f position;
+    /* 0x1C */ Vec3f acceleration;
+    /* 0x28 */ Vec3f velocity;
+    /* 0x34 */ f32 scaleXY;
+    /* 0x38 */ f32 scaleXYDelta;
+} EnGoEffect; // size = 0x3C;
 
 typedef struct EnGo {
     /* 0x000 */ Actor actor;
     /* 0x144 */ SkelAnime skelAnime;
     /* 0x188 */ EnGoActionFunc actionFunc;
-    /* 0x18C */ EnGoActionFunc unk_18C;
-    /* 0x190 */ EnGoActionFunc unk_190;
+    /* 0x18C */ EnGoActionFunc interruptedActionFunc;
+    /* 0x190 */ EnGoActionFunc graveBroDialogActionFunc;
     /* 0x194 */ ColliderCylinder colliderCylinder;
     /* 0x1E0 */ UNK_TYPE1 unk1E0[0x4C];
```

Comment:
Hmm, this certainly looks like another cylinder. I'd have no problem with typing it as such and calling it `unusedCylinder`.